vsdkx/addon/tracking/settings.py

```diff
@@ -0,0 +1,7 @@
+DEFAULT = {
+    "bidirectional_mode": True,
+    "bidirectional_threshold": 150,
+    "distance_threshold": 500,
+    "max_disappeared": 10,
+    "min_appearce": 40
+}
```

Comment:
Also add the `debug_mode` attribute.
```suggestion
     "debug_mode": False
}
```

Reply:
it's wrong place for this. debug mode is not turned on from addon settings. it should be defined in model settings and gonna be read by core. this is read by tracking addon only, which doesn't need it